src/Mod/TechDraw/App/AppTechDraw.cpp

```diff
@@ -121,7 +121,7 @@ PyMOD_INIT_FUNC(TechDraw)
     TechDraw::GeomFormat            ::init();
     TechDraw::PropertyCenterLineList::init();
     TechDraw::CenterLine            ::init();
-    TechDraw::PropertyCosmeticEdgeList::init();

```

Comment:
Can not remove PropertyCosmeticEdgeList as old document files will contain references to this.  Need to keep this around so old documents restore properly.